content/github/managing-your-work-on-github/linking-a-repository-to-a-project-board.md

```diff
@@ -30,6 +26,12 @@ Anyone with write permissions to a project board can link repositories owned by
 7. Click **Link**. To unlink, click **Unlink**.
 ![Link button](/assets/images/help/projects/link-button.png)
 
+{% note %}
+
+**Note:** In order to link a repository to your organization or user owned project board the repository needs to have issues enabled (In forked repositories issues are disabled by default).  "For information on how to see if issues are enabled or disabled for a repository, see "[Disabling issues for a repository](/github/managing-your-work-on-github/disabling-issues) ."
```

Comment:
Thanks for the changes to the note. I'm going to suggest a couple of small punctuation changes. Also, I think it would be helpful to mention the easy way to see whether issues are enabled for a repository, as suggested in the content design plan.

```suggestion
**Note:** In order to link a repository to your organization or user owned project board the repository needs to have issues enabled. That is, the repository has an "Issues" tab (in forked repositories issues are disabled by default).  For information on how to enable or disable issues for a repository, see "[Disabling issues for a repository](/github/managing-your-work-on-github/disabling-issues)."
```